src/__swaps__/screens/Swap/components/GasPanel.tsx

```diff
@@ -442,14 +453,14 @@ export function GasPanel() {
   });
 
   return (
-    <Box as={Animated.View} paddingHorizontal="12px" zIndex={12} style={[styles, { flex: 1 }]} testID="gas-panel" width="full">
+    <Box as={Animated.View} paddingHorizontal="12px" zIndex={12} style={[styles]} testID="gas-panel" width="full" onLayout={onLayout}>
       <Stack alignHorizontal="center" space="28px">
         <Text weight="heavy" color="label" size="20pt">
           {i18n.t(i18n.l.gas.gas_settings)}
         </Text>
 
-        <Box gap={24} width="full" alignItems="stretch">
-          <Box gap={24} height="104px">
+        <Box gap={24} width="full" alignItems="stretch" style={{ marginBottom: 30 }}>
```

Comment:
Where does the number 30 come from here? Also this is a nit it's the same either way but I would do `marginBottom={{ custom: 30 }}` for props that support that format over adding a style prop.

Reply:
30 is arbitrary here to get the content to look good, other change has been made.